Core/clim-core/presentations.lisp

```diff
@@ -1310,6 +1310,7 @@ and used to ensure that presentation-translators-caches are up to date.")
               ,@body)))
         (t
          `(lambda (,(car args) &key ,@(cdr args) &allow-other-keys)
+            (declare (ignorable ,(car args)))
             ,@body))))
```

Comment:
whet about the key arguments? Is it possible that they are not referenced in body? Then it would be
`(declare (ignorable ,@args))`